flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/crd/sessionjob/status/FlinkSessionJobStatus.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.operator.crd.sessionjob.status;
+
+import org.apache.flink.annotation.Experimental;
+import org.apache.flink.kubernetes.operator.crd.status.JobStatus;
+import org.apache.flink.kubernetes.operator.observer.JobManagerDeploymentStatus;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/** Last observed status of the Flink Session job. */
+@Experimental
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class FlinkSessionJobStatus {
+
+    /** Last observed status of the job. */
+    private JobStatus jobStatus = new JobStatus();
+
+    /** Last observed status of the JobManager deployment. */
```

Comment:
Do we really need the `jobManagerDeploymentStatus` here? Or we could get it from the session `FlinkDeployment.status`.

Reply:
We could get from the `FlinkDeployment.status`, Removing this field.